onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -392,12 +392,12 @@ bool CheckIsInitializer(const InitializedTensorSet& initializers, const NodeUnit
   return true;
 }
 
-std::vector<int32_t> OnnxIndexesToNnapi(gsl::span<const int64_t> onnx_indexes, std::optional<size_t> input_size = std::nullopt) {
+std::vector<int32_t> OnnxAxesToNnapi(gsl::span<const int64_t> onnx_axes, std::optional<size_t> input_rank = std::nullopt) {
```

Comment:
thanks for renaming it. we should probably move the default argument value for `input_rank` to helper.h so it is more visible.